test/framework/src/main/java/org/elasticsearch/test/AbstractQueryTestCase.java

```diff
@@ -399,6 +399,7 @@ public void testToQuery() throws IOException {
             QB controlQuery = copyQuery(firstQuery);
             setSearchContext(randomTypes, context); // only set search context for toQuery to be more realistic
             Query firstLuceneQuery = rewriteQuery(firstQuery, context).toQuery(context);
+            assertNotNull("toQuery should not return null", firstLuceneQuery);
```

Comment:
shall we do the same with the second lucene query and other possible queries we have here?